src/components/[guild]/EditGuild/components/Events/EventInput.tsx

```diff
@@ -29,34 +31,46 @@ const logos: Record<EventSourcesKey, string> = {
 }
 
 const EventInput = ({ eventSource }: Props) => {
-  const { register, setValue } = useFormContext<GuildFormType>()
+  const {
+    register,
+    setValue,
+    formState: { errors },
+  } = useFormContext<GuildFormType>()
+
   const link = useWatch({
     name: `eventSources.${eventSource}`,
   })
 
   if (!link && link !== "") return null
```

Comment:
I know this isn't a new line, but why do we need this exactly?

Reply:
If the link is null or undefined, we don't display the input. However, the link is an empty string which means the user wants to add a new link.

Reply:
ah okay, right, thanks!